ui/src/components/inputs/MonacoEditor.vue

```diff
@@ -463,9 +473,31 @@
 
     const disposeCompletions = ref<() => void>();
 
+
+    const errorMarkers = ref<any[]>([]);
+
+    function updateErrorMarkers() {
+        const model = localEditor?.getModel();
```

Comment:
Same here, this line is producing console error an preventing me to continue. Could you please amend it and make sure there are no more errors in the console regarding this change?

![image](https://github.com/user-attachments/assets/d1a54966-34cf-4b5a-8564-0f52b4585e4f)